tenable/sc/organization.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
+"""
+from tenable.sc.base import SCEndpoint
+
+
+
+class OrganizationAPI(SCEndpoint):
+
+    def _constructor(self, **kw):
+        '''
+        Organization document constructor
+        '''
+        if 'name' in kw:
+            kw["name"] = self._check('name', kw['name'], str)
+
+        if 'description' in kw:
+            kw['description'] = self._check('description', kw.get('description', ''), str)
+
+        if 'address' in kw:
+            kw['address'] = self._check('address', kw.get('address', ''), str)
+
+        if 'city' in kw:
+            kw['city'] = self._check('city', kw.get('city', ''), str)
+
+        if 'state' in kw:
+            kw['state'] = self._check('state', kw.get('state', ''), str)
+
+        if 'country' in kw:
+            kw['country'] = self._check('country', kw.get('country'), str)
+
+        if 'phone' in kw:
+            kw['phone'] = self._check('phone', kw.get('phone'), str)
+
+        if 'lces' in kw:
+            kw["lces"] = self._check('lces', kw['lces'], list, default=[])
+
+        if 'zone_selection' in kw:
+            kw["zoneSelection"] = self._check(
+                'zone_selection', kw['zone_selection'], str,
+                choices=["auto_only", "locked", "selectable",
+                         "selectable+auto", "selectable+auto_restricted"])
+            del kw['zone_selection']
```

Comment:
use the del() function instead of the del statement for portability

Reply:
done, also for the rest `del` function